autopush/tests/test_main.py

```diff
@@ -180,6 +180,7 @@ class test_arg:
         message_write_throughput = 0
         senderid_list = '{"12345":{"auth":"abcd"}}'
         s3_bucket = "none"
+        key_hash = ""
```

Comment:
This evaluates to False, so the code checking for key_hash will not trigger, which is why the code coverage dropped.